app/lib/katello/validators/content_view_environment_validator.rb

```diff
@@ -6,14 +6,27 @@ def validate(record)
         environment_id = record.respond_to?(:lifecycle_environment_id) ? record.lifecycle_environment_id : record.environment_id
 
         if record.content_view_id && environment_id
-          view = ContentView.find(record.content_view_id)
-          env = KTEnvironment.find(environment_id)
-          unless view.in_environment?(env)
+          view = rescue_not_found(record, 'content_view', record.content_view_id) do
+            ContentView.find(record.content_view_id)
+          end
+          env = rescue_not_found(record, 'environment', environment_id) do
+            KTEnvironment.find(environment_id)
+          end
+          unless record.errors[:content_view_id].empty? ||
+              record.errors[:environment_id].empty? ||
+              view.in_environment?(env)
             record.errors[:base] << _("Content view '%{view}' is not in environment '%{env}'") %
                                       {:view => view.name, :env => env.name}
           end
         end
       end
+
+      def rescue_not_found(record, resource, id)
```

Comment:
Had to add this because some tests like this https://github.com/Katello/katello/pull/7064/files#diff-37eee8f11c299eac974ae0eea557ddb0L46 are trying to create ContentFacets without ContentView/Environment. 

Since `content_view_id` and `environment_id` default to 1, this validator tries to find these resources with ID 1, which isn't available in the testing env. 

```
```